docker/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+docker  buildx \
+        build \
+        --progress plain \
+        --platform linux/amd64 \
+        -t esc-configurator:latest \
+        .
+
+docker  run \
+        -i \
+        -d --publish 1234:1234 \
```

Comment:
Would be cool if the port would be a optional parameter to the script itself, or pull it from an env variable.

Reply:
Fixed.